ironfish-cli/src/commands/start.ts

```diff
@@ -110,6 +110,11 @@ export default class Start extends IronfishCommand {
       description:
         'Path to a JSON file containing the network definition of a custom network to connect to',
     }),
+    enableWallet: Flags.boolean({
```

Comment:
```suggestion
    wallet: Flags.boolean({
```

`--no-enableWallet` reads a bit weird. Let's make this a little cleaner